### PR TITLE
[MetadataExtractor] Add a StitchedContent360 metadata API

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add a StitchedContent360 metadata API

### API Changes ###

 - TCSACR-594

### CODE REVIEW

File: src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs [hunk 1]
```diff
@@ -402,6 +403,16 @@ internal Metadata(MetadataExtractor extractor)
...
         public string Content360 { get; }
 
+        /// <summary>
+        /// Gets the information for stitched 360 content of the media.
+        /// </summary>
+        /// <value>
+        /// 0:None(There's no stitched 360 content metadata), 1:Non-stitched content, 2:stitched content
+        /// </value>
+        /// <since_tizen> 12 </since_tizen>
+        /// <value>A string representing the information for stitched 360 content, or null if the information does not exist.</value>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs:413]: The `<value>` tag has been added twice, please use only one.

File: src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs [hunk 2]
```diff
@@ -402,6 +403,16 @@ internal Metadata(MetadataExtractor extractor)
         /// <value>A string representing the information for 360 content, or null if the information does not exist.</value>
         public string Content360 { get; }
 
+        /// <summary>
+        /// Gets the information for stitched 360 content of the media.
+        /// </summary>
+        /// <value>
+        /// 0:None(There's no stitched 360 content metadata), 1:Non-stitched content, 2:stitched content
```
Comment (wiertel) [inline src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs:410]: is it 0 or None or maybe both for cases when content does not specify that information?
Could we have an enum with the listed possible values? That would make the description easier.  Or maybe nullable int (`int?`) could be used? That would also depend on underlying metadata stored.

File: src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs [hunk 3]
```diff
@@ -402,6 +403,16 @@ internal Metadata(MetadataExtractor extractor)
         /// <value>A string representing the information for 360 content, or null if the information does not exist.</value>
         public string Content360 { get; }
 
+        /// <summary>
+        /// Gets the information for stitched 360 content of the media.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs:407]: How does it work? A 360 video can be stitched or not, and this does not apply to any other media, right? Maybe a description like "Gets the information about 360 video content being stitched or not." or "Provides information whether a 360 video content is stitched."

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs:413]: I'll fix it.

File: src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs (hunk 3 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs:407]: I'll change it as "Gets the information about 360 video content being stitched or not."

File: src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs:410]: I'll change it as `int?` type.

**Review by wiertel — APPROVED** [review]

**Pull request merged**